nixos/modules/config/fonts/fontconfig.nix

```diff
@@ -339,17 +344,10 @@ in
 
   };
   config = mkIf cfg.enable {
-    fonts.fontconfig.confPkgs = [ confPkg ];
+    fonts.fontconfig.confPkgs    = [ confPkg ];
```

Comment:
A nitpick: we usually name options like those `*Packages`, not `*Pkgs` -- for example, `udev.packages`, `environment.systemPackages`, `boot.kernelPackages` etc.